Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla43941.cs

```diff
@@ -33,12 +33,10 @@ public void Bugzilla43941Test()
 				RunningApp.Back();
 			}
 
-			// At this point, the counter can be any value, but it's most likely not zero.
 			// Invoking GC once is enough to clean up all garbage data and set counter to zero
 			RunningApp.WaitForElement(q => q.Marked("GC"));
 			RunningApp.Tap(q => q.Marked("GC"));
-
-			RunningApp.WaitForElement(q => q.Marked("Counter: 0"));
+			Assert.AreEqual(0, LandingPage43941.Counter);
```

Comment:
The UITest and the App are different processes and so each get their own copy of `.Counter`; The UITest needs to use `RunningApp` to query the `.Counter`'s value in the other process. 

The app exposes the `.Counter` value by updating the `Label`'s text so that the UITest can query `"Counter: 0"`.